symplyphysics/definitions/acceleration_is_velocity_derivative.py

```diff
@@ -18,15 +18,15 @@
 :attr:`~symplyphysics.symbols.kinematic.acceleration` of the body as a function of time.
 
 Symbol:
-    :code:`a`
+    :code:`a = a(t)`
```

Comment:
Why not simply `a(t)`?

Reply:
I thought that in that case I would need to make the law look something like `a(t) = d(v(t))/dt` for consistency, where function application seems redundant. Can leave those simply as `a(t)` of course.

Reply:
Yes, for consistency they should be updated in laws as well.